intent-classifier/training/train_intent.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+const path = require('path');
+const fs = require('fs');
+
+const argparse = require('argparse');
+const tf = require('@tensorflow/tfjs');
+const fileIO = require('@tensorflow/tfjs-node/dist/io/file_system');
+const mkdirp = require('mkdirp');
+
+const {getModel} = require('./intent_model');
+const {loadJSON} = require('./util');
+
+
+async function run(
+    dataPath, metadataPath, outputFolder, epochs, validationSplit = 0.15) {
+  const {xsArr, ysArr} = loadJSON(dataPath);
+  const metadata = loadJSON(metadataPath);
+
+  const xs = tf.tensor(xsArr, metadata.xsShape);
+  const ys = tf.tensor(ysArr, metadata.ysShape);
+
+  const model = getModel(metadata.labels);
+
+  await model.fit(xs, ys, {epochs, validationSplit});
```

Comment:
noticed you are using `model.fit` here but `model.fitDataset` in train_tagger. Can you switch to fitDataset for consistency?